guides/plugins/plugins/administration/system-updates/vite.md

```diff
@@ -22,6 +22,45 @@ The Vue.js ecosystem has built its own bundler: Vite. Vite is fast, easier to co
 
 For apps there are no consequences as your build process is already decoupled from Shopware. For plugins you only need to get active if you currently extend the webpack config by providing your own `webpack.config.js` file.
 
+### Migrate the custom webpack config to Vite
+
+If you have a custom webpack config, you need to migrate it to Vite. You need to do the following steps:
+
+  1. Create a new config file `vite.config.mts` to your plugin in the `YourApp/src/Resources/app/administration/src` directory. Previously you had a `webpack.config.js` one folder above.
```

Comment:
That's btw not true. Previously the webpack.config.js needed to be in `YourApp/src/Resources/app/administration/build`